pages/api/Schema/reports/fourteenMonth/fourteenMonth.graphql

```diff
@@ -41,7 +41,7 @@ type FourteenMonthReportContact {
   total: Float!
   average: Float!
   minimum: Float!
-  months: [FourteenMonthReportContactMonths!]
+  months: [FourteenMonthReportContactMonths!]!
```

Comment:
Does the API give us back an empty array or something, guaranteeing never to be null or missing from the response?

Reply:
As far as I can tell, `summarize_months` has to return an array because if `group_donations_by_month` doesn't return an array, the `.map` would throw https://github.com/CruGlobal/mpdx_api/blob/f1f4f35b42bef9483e1eff0d3c8f4003bc111437/app/services/reports/donor_currency_donations.rb#L58-L69

For what it's worth, on the JS side, if `months` ever were an array, this line would throw too. The proxy endpoint couldn't actually handle null months before.
https://github.com/CruGlobal/mpdx-react/blob/b35116164f7a6b9ba6cdcb6b75a9f5d75eb9ee0f/pages/api/Schema/reports/fourteenMonth/datahandler.ts#L114